es6-classes/circle.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(radius) {
+    super(radius);
+    this.radius = radius;
+    this.area = Math.PI * this.radius * (Math.PI * this.radius);
+    this.perimeter = 2 * Math.PI * radius;
```

Comment:
You should not be creating these properties manually. When you call `super()` it will generate these for you. So utilizing your `radius` parameter, you need to calculate what `area` or `perimeter` would be, and that's what you pass into `super`